src/pudl/extract/eia860m.py

```diff
@@ -51,6 +52,19 @@ def process_raw(self, df, page, **partition):
         df = fix_leading_zero_gen_ids(df)
         return df
 
+    def extract(self, settings: Eia860Settings = Eia860Settings()):
+        """Extracts dataframes.
+
+        Returns dict where keys are page names and values are
+        DataFrames containing data across given years.
+
+        Args:
+            settings (Eia860Settings): Object containing validated settings
```

Comment:
We don't need the argument types in the docstring when they're specified by a type hint -- Sphinx and many IDEs will pull the type from the hint directly.

Reply:
Fixed